resources/camino-creator/main.scss

```diff
@@ -2,18 +2,26 @@
 @import url("https://fonts.googleapis.com/css?family=Nunito");
 
 // Bootstrap
-@import "~bootstrap/scss/bootstrap";
-
-html {
-  height: 100%;
-}
+@import "~bootstrap/dist/css/bootstrap.min.css";
```

Comment:
curious - what's the reason for pulling the CSS vs the SASS? (I usually pull the SASS so we can override their variables if desired)

Reply:
> I usually pull the SASS so we can override their variables if desired

[Oh – I know...](https://github.com/UMN-LATIS/Camino/blob/main/resources/sass/app.scss#L6) 😉

No real reason though, other than I was troubleshooting and this is where things landed.